### PR TITLE
made my edit pass

### PR DESCRIPTION
**Pull request by dschride**

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -89,19 +91,21 @@ \section*{Introduction}
 groups. Beyond a lack of efficiency from duplicated effort, such a situation leads
 to decreases in reproducibility and transparency across the entire field.
 Here we describe \stdpopsim, a community-driven catalog
-of standardized simulations across a variety of common study organisms
-that will empower the field and usher in the next era of genetics data analysis.
+of standardized simulations spanning a variety of common study organisms.
+This resource will empower the field to rigorously evaluate population genetic software
+and select the best tool for a given task, thereby helping to usher in the next era of population genetic data analysis.
```
Comment (andrewkern) [inline popSimManu.tex:96]: i'm going leave this in but think we should edit it further

File: popSimManu.tex [hunk 2]
```diff
@@ -149,18 +153,18 @@ \section*{Introduction}
 This resource, which we call \texttt{stdpopsim}, makes running
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community maintained catalog.
-The \stdpopsim catalog currently contains four organisms: humans,
+The \stdpopsim catalog currently contains five organisms: humans, orangutans
```
Comment (andrewkern) [inline popSimManu.tex]: pongo is still out. will edit on your branch before the merge

**Review by andrewkern — APPROVED** [review]
lgtm @dschride. i pushed a few minor edits to your branch and then will merge.

File: popSimManu.tex (hunk 2 above)
Comment (dschride) [reply popSimManu.tex]: Okay but it is mentioned elsewhere so be sure to take that out.

Comment (andrewkern) [reply popSimManu.tex]: let's get some more clarity on this. i'll raise an issue.

**Pull request merged**